src/components/Map/Map.jsx

```diff
@@ -7,14 +7,17 @@ import "leaflet-defaulticon-compatibility";
 import {mapResults} from "@/components/Map/mockResults";
 import {ItemMarker} from "@/components/Map/ItemMarker";
 import L from "leaflet";
+import {SelectionOverlay} from "@/components/Map/SelectionOverlay";
+import {useChatMutation} from "@/store/services/chatApi";
 
 export const Map = (props) => {
   const {position: initialPosition} = props;
   const icon = L.icon({iconUrl: "/images/marker-icon.png"});
-
+  const [chat, { isLoading, isSuccess, data, error }] = useChatMutation();
+  console.log(data);
   const [position, setPosition] = useState(initialPosition); // State to store user's position
   const [loading, setLoading] = useState(true);  // Loading state for geolocation
-
+  console.log(isLoading, isSuccess, error, data);
   useEffect(() => {
```

Comment:
This geolocation fetching is redundant, as parent component fetches it and passes it to map